feature-libs/product-configurator/rulebased/core/connectors/rulebased-configurator.adapter.ts

```diff
@@ -14,10 +14,12 @@ export abstract class RulebasedConfiguratorAdapter {
    * Abstract method used to create a configuration
    *
    * @param productCode Root product code
+   * @param forceReset optional - if true, creation of a new default configuration will be enforced, instead of returning a product bound configuration
    */
   abstract createConfiguration(
     owner: CommonConfigurator.Owner,
-    configIdTemplate?: string
+    configIdTemplate?: string,
+    forceReset?: boolean
```

Comment:
@Uli-Tiger just FYI: It seems the BC tool is smart enough to know that this is not breaking. Was not sure about that, but I didn't get a warning also for the previous change (adding configIdTemplate)